src/SSW.SophieBot/SSWSophieBot/dialogs/CheckAvailabilityDialog/language-generation/en-us/CheckAvailabilityDialog.en-us.lg

```diff
@@ -54,7 +54,9 @@
     - is free.
 
 # ClientInfo(profile)
-- IF: ${ profile.bookingStatus == 1 && profile.clients.count != 0 }
+- IF: ${!exists(user.TenantAuthenticated) || user.TenantAuthenticated == false}
+    - **Client**: \\\\*\\\\*\\\\*

```

Comment:
Profile cards also have client info, not sure if you include that.

Reply:
Done